### PR TITLE
cc-toaster: use @lit-lab/motion to implement animation

### LINKED ISSUE #549: Experiment with lit labs motion (animation) for the toaster
The `cc-toaster` component has some animations that could maybe benefit from lit labs' motion project: https://github.com/lit/lit/tree/main/packages/labs/motion

### PR DESCRIPTION
**Pull request by pdesoyres-cc**
Fixes #549

### CODE REVIEW

**Comment by pdesoyres-cc** [comment]
oh noo :sob: https://github.com/runem/lit-analyzer/issues/218

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "components:build-cdn:versions-list": "node tasks/update-versions-list.js",
     "components:publish-cdn": "node tasks/publish-cdn.js",
     "components:check-i18n": "node tasks/check-i18n.js",
-    "components:check-lit": "lit-analyzer src",
+    "components:check-lit": "lit-analyzer --rules.no-unknown-attribute off src",
```
Comment (hsablonniere) [inline package.json:25]: This should be configured in the `tsconfig.json` at the root of the projet.
But even if it's a bit weird, I'd prefer to use the `globalAttributes` config, see https://github.com/runem/lit-analyzer/blob/master/docs/readme/config-table.md

I tried this and it works :scream: 

```json
{
   ...
  "globalAttributes": [
    "________________________________",
    "___________________________________________________________"
  ],
  ...
}

**Review by hsablonniere — CHANGES_REQUESTED** [review]
Hey @pdesoyres-cc 

Well done on this one. The result is really nice. No more jumps when toasts arrive.

Nice idea to create a reactive controller for the media query but in reality, prefer reduce motion doesn't really change. I would prefer to remove it and simplify the code by just calling match in the render.

File: package.json (hunk 1 above)
Comment (pdesoyres-cc) [reply package.json:25]: haha! nice hack!

**Review by hsablonniere — APPROVED** [review]

File: tsconfig.json [hunk 2]
```diff
@@ -39,6 +39,10 @@
           "animationend",
           "mouseenter",
           "mouseleave",
+        ],
+        "globalAttributes": [
+          "________________________________",
+          "___________________________________________________________"
```
[the remark this answers is not in the document]
Comment (florian-sanders-cc) [reply tsconfig.json:45]: Thanks !

**Review by florian-sanders-cc — APPROVED** [review]
Gg this is so cool ! :sunglasses:

**Comment by hsablonniere** [comment]
@pdesoyres-cc You can squash the "feedback from Hubert" and merge it yourself when you're ready :wink: 
Well done :clap:

**Pull request merged**